selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/RenameWorkspaceTest.java

```diff
@@ -85,14 +86,14 @@ private void renameWorkspace(String name) {
     dashboardWorkspace.enterNameWorkspace(name);
     assertFalse(dashboardWorkspace.isWorkspaceNameErrorMessageEquals(WS_NAME_TOO_SHORT));
     assertFalse(dashboardWorkspace.isWorkspaceNameErrorMessageEquals(WS_NAME_TOO_LONG));
-    clickOnSaveButton();
+    clickOnSaveBtnAndWaitWorkspaceRestarted();
     dashboardWorkspace.checkNameWorkspace(name);
   }
 
-  private void clickOnSaveButton() {
+  private void clickOnSaveBtnAndWaitWorkspaceRestarted() {
```

Comment:
I think **saveAndWaitWorkspaceRestarted()** would be better name so as describe whole method propose dispite of concrete steps realizations.